tsconfig.json

```diff
@@ -13,7 +13,8 @@
     "resolveJsonModule": true,
     "isolatedModules": true,
     "jsx": "preserve",
-    "incremental": true
+    "incremental": true,
+    "baseUrl": "."
```

Comment:
I'm guessing you added this to remove the need for absolute path imports?

Reply:
yes